src/test/java/org/kuali/kfs/sys/fixture/UserNameFixture.java

```diff
@@ -46,6 +46,7 @@
 ls35,
 // favorite account manager
 db18,
```

Comment:
I would probably make sense to remove it. It is however used by another unit test so that would need to be cleaned up as well.